script.js

```diff
@@ -11,7 +11,22 @@ const setData = data =>
 	window.localStorage.setItem("APIdata", JSON.stringify(data));
 const getData = key => JSON.parse(window.localStorage.getItem(key));
 
-setData(db);
+const demoData = () => {
+	setData(db);
+};
+const clearData = () => {
+	window.localStorage.clear();
```

Comment:
That's not a good idea as we have a key under which our "database" sits and now we just purge it all out.